www/templates/www/base.html

```diff
@@ -3,93 +3,125 @@
-              {% else %}
-                <a class="nav-link" href="{% url 'login' %}">{% trans 'Log in' %}</a>
-              {% endif %}
-            </li>
-            <li class="nav-item">
-              <form id="langform" class="form-inline" action="{% url 'set_language' %}" method="post">{% csrf_token %}
-                <div class="input-group input-group-sm">
+          </li>
+          <li class="nav-item">
+            <form id="langform" class="form-inline" action="{% url 'set_language' %}" method="post">{% csrf_token %}
+              <div class="input-group input-group-sm">
                 <div class="input-group-prepend">
                   <span class="input-group-text" id="inputGroup-sizing-sm">{% trans 'Language' %}</span>
                 </div>
                 <select class="custom-select" name="language" id="language-select">
-                {% get_current_language as LANGUAGE_CODE %}
-                {% get_available_languages as LANGUAGES %}
-                {% get_language_info_list for LANGUAGES as languages %}
-                {% for language in languages %}
-                  <option value="{{ language.code }}"{% if language.code == LANGUAGE_CODE %} selected{% endif %}>
+                  {% get_current_language as LANGUAGE_CODE %}
+                  {% get_available_languages as LANGUAGES %}
+                  {% get_language_info_list for LANGUAGES as languages %}
+                  {% for language in languages %}
+                  <option value="{{ language.code }}" {% if language.code==LANGUAGE_CODE %} selected{% endif %}>
                     {{ language.name_local }}
                   </option>
-                {% endfor %}
+                  {% endfor %}
                 </select>
-                </div>
-              </form>
-            </li>
-          </ul>
-        </div>
-      </nav>
-    </header>
+              </div>
+            </form>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
 
-    <main role="main" class="container-fluid">
+  <main role="main" class="container-fluid">
     {% block content %}{% endblock content %}
-    </main>
+  </main>
 
-    <footer class="footer">
-      <div class="container">
-        <span class="text-muted">
-          <a target="_blank" rel="noopener noreferrer" href="{{ PRIVACY_POLICY_URL }}">{% trans 'Privacy policy' %}</a>
-          {% trans 'Report bugs at' %}: <a target="_blank" rel="noopener noreferrer" href="{{ GITHUB_URL }}">{% trans 'GitHub' %}</a>
-          | <a href="{% url 'changelog-view' %}">{% trans 'View changelog' %}</a>
-        </span>
-      </div>
-    </footer>
+  <footer class="footer">
+    <div class="container">
+      <span class="text-muted">
+        <a target="_blank" rel="noopener noreferrer" href="{{ PRIVACY_POLICY_URL }}">{% trans 'Privacy policy' %}</a>
+        {% trans 'Report bugs at' %}: <a target="_blank" rel="noopener noreferrer" href="{{ GITHUB_URL }}">{% trans
+          'GitHub' %}</a>
```

Comment:
django does not like this kind of newlines inside `trans` tags. I'm guessing your editor did some automatic styling for the file that will not work